frontend/app/src/views/settings/DefiSettings.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <v-card>
+    <v-card-title>Defi Settings</v-card-title>
+    <v-card-text>
+      <h3>Active Defi modules</h3>
+      <p class="text--secondary">
+        Only data for the selected modules will be queried. If no modules are
+        selected, querying for the supported modules will be disabled.
```

Comment:
maybe specify: 

querying for the supported modules -> querying for **all** the supported modules